chrome/content/zotero/zoteroPane.js

```diff
@@ -538,6 +538,12 @@ var ZoteroPane = new function()
 			if (state) {
 				Zotero_Tabs.restoreState(state.tabs);
 			}
+			setTimeout(function () {
```

Comment:
Without this timeout, there is an error either that collectionTree is not loaded yet or that the item for that note is not loaded. Is there a better way? Some kind of hook that fires when everything is 100% loaded and ready?